packages/libs/react-ui/src/components/Typography/typography.css.ts

```diff
@@ -1,20 +1,20 @@
-/* eslint @kadena-dev/typedef-var: 0 */
-
 import { sprinkles } from '../../styles';
 
-export const fontVariants = {
-  main: [sprinkles({ fontFamily: 'main' })],
-  mono: [sprinkles({ fontFamily: 'mono' })],
+import { ComplexStyleRule } from '@vanilla-extract/css';
+
+export const fontVariants: Record<string | number, ComplexStyleRule> = {
+  main: [sprinkles({ fontFamily: '$main' })],
+  mono: [sprinkles({ fontFamily: '$mono' })],
 };
 
-export const transformVariants = {
+export const transformVariants: Record<string | number, ComplexStyleRule> = {
   uppercase: [sprinkles({ textTransform: 'uppercase' })],
   lowercase: [sprinkles({ textTransform: 'lowercase' })],
   capitalize: [sprinkles({ textTransform: 'capitalize' })],
   none: [sprinkles({ textTransform: 'none' })],
 };
 
-export const colorVariants = {
-  default: [sprinkles({ color: 'neutral4' })],
-  emphasize: [sprinkles({ color: 'neutral6' })],
+export const colorVariants: Record<string | number, ComplexStyleRule> = {
```

Comment:
Can you give an example of when type `number` is need?

Reply:
I just copied over the libraries prop type, but there is no need for the number in this case. I will remove it 👍